articles/support/tickets.md

```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
+1. From [Support Center](${env.DOMAIN_URL_SUPPORT}), click on the **Open Ticket** button. If you don't see this button, then you do not have access to support (only paying and trial customers have access to Support). In this case you can use our [Community](https://community.auth0.com/) instead (click **Ask our Community**).
 ![Support Center](/media/articles/support/open-ticket.png)
-1. Choose the **Affected Account** from the dropdown menu.
-1.  Under **What can we help you with?** select the type of issue that best fits your case.
+1. Choose the **Affected Tenant** from the dropdown menu.
+1. Under **Issue Type** select the type of issue that best fits your case.
+* Public Cloud Support Incident
+* Appliance Support Incident 
+* Enhancement/Feature Request
+* Billing or Payment 
+* Compliance/Legal
 ![Issue types](/media/articles/support/issue-types.png)
     * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
         * With Service Issues, you will be asked to select a severity:
             *  **Low: Product Question** - You have questions about how things work, whether we support something
             * **Normal: General Support Question** - You are in development and have a question about how to configure something or how to resolve an error
             * **High: Production Application Issue** - Your Production application is experiencing a minor outage impacting some users/feature(s)
             * **Urgent: Production Application Offline** - Your Production application is experiencing a critical outage impacting all users
-        *  You will then be asked to choose an option under "What is the service issue related to?"
-            *  _You may be prompted for additional information depending on your selection._
+        *  Depending on the issue type you chose after you select the severity, you may need to answer the **What can we help you with?**. Choose one of the following:
+            * **Something is broken**
+            * **How to** 
+            * **Enhancement/Feature Request**
+            * **Billing or Payment**
+            * **Compliance/Legal**
+            * **Outage**
+            * **Do not understand self-service resources**
+            * **Enterprise Connections**
+            * **SDK/Library**
+            * **Management API**
+            * **Custom DB Connection**
+            * **SSO Integrations**
+![Customer Reason](/media/articles/support/customer-reason.png)
+        * After you make your selection, you can choose to answer some follow-up questions to further refine your request. You can also choose to skip this step by clicking the toggle. 
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
```

Comment:
We need to add here the other options for issue type:
* Enhancement/Feature Request . -> in replacement of "Auth0 Service Feedback"
* Billing or Payment -> in replacement of "Billing, Payments, or Account Issues"
* Compliance/Legal -> in replacement of "Vulnerability Reports or Legal Issues"

Notice that the "Other" option was removed.

Reply:
Those three were added in lines 41-43.

Reply:
yes, but noticed the option names changed @tamigoodall !
<img width="1001" alt="issuetypeoptions" src="https://user-images.githubusercontent.com/1350396/44293674-ec04ce00-a263-11e8-8aee-096ccf986880.png">

Reply:
Fixed.